server/models/video/video.ts

```diff
@@ -597,6 +598,15 @@ export class VideoModel extends Model<Partial<AttributesOnly<VideoModel>>> {
   })
   VideoPlaylistElements: VideoPlaylistElementModel[]
 
+  @HasMany(() => VideoSourceModel, {
```

Comment:
I'm not sure to understand how a video may have multiple source files

Reply:
If uploading a new file it would be possible to store the old version while transcoding the new. Theoretically it would also be possible to save the old version to let the user be able to rollback to the old version, I'm not sure if there's any use case for that though.

Reply:
I prefer to not design the database according to a potential future use case. So let's keep things simple and just have 1:1 relation between these tables.